src/SlnGen.Build.Tasks/ExtensionMethods.cs

```diff
@@ -106,9 +106,9 @@ public static IEnumerable<string> GetConditionedPropertyValuesOrDefault(this Pro
         /// <returns>The values of the property if one exists, otherwise the default value specified.</returns>
         public static IEnumerable<string> GetPossiblePropertyValuesOrDefault(this Project project, string name, string defaultValue)
         {
-            var propertyValue = project.GetPropertyValue(name);
-            IEnumerable<string> conditionPropertyValues = project.GetConditionedPropertyValuesOrDefault(name, string.Empty).ToList();
-            List<string> values = new List<string>();
+            HashSet<string> values = new HashSet<string>(StringComparer.OrdinalIgnoreCase);
```

Comment:
Wanted to maintain the actual order, but turned out VS doesn't care and sorts them alphabetically